src/touch/gestures.c

```diff
@@ -59,11 +61,16 @@ typedef struct {
     uint16_t position_start;
     // The last measured position.
     uint16_t position_current;
+    /*
+     * The maximum distance the finger has travelled
+     * since starting the gesture.
+     */
+    uint16_t max_slide_travel;
     int32_t velocity_sum;
     uint16_t velocity_index;
     // Stores the velocities.
     int16_t velocity_values[MAX_HISTORY];
-    // The gesture type (tap, slide).
+    // The gesture type (tap, slide). FUTURE: remove this?
```

Comment:
Looks like the changes below allow TAP and SLIDE to be independent of each other (i.e. can occur at the same time). Then best to remove gesture_type as its existence implies only one state at a time (i.e. will be confusing).

Reply:
It is confusing but it is what the slide processing is currently relying on. Better not to push further functional changes atm. Maybe I can open a separate issue?